README.md

```diff
@@ -49,11 +60,11 @@ Ensure that you meet the following prerequisites before you use Explorer for End
 
 **Host-side prerequisites**:
 
-- Endevor version 18.0.12 or higher with the following PTFs applied:
```

Comment:
That comma after the number is redundant

Reply:
Could you specify exactly where is that comma? I can see it only in this line: `(v18.0.x only) PTFs SO09580, SO09581, SO10013, and SO11268`, but the entire line was completely removed.

Reply:
I'm sorry, this should be discarded since you've removed lines 52-56.